src/components/SearchHint/SearchHint.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './SearchHint.scss';
+
+export const SearchHint: React.FC<{ hint: string }> = ({ hint }) => (
+  <Link className="searchHint__link" to={`/profile/${hint}`}>
```

Comment:
In my opinion instead of hint variable name we should uid or something that is going to identify user. I think it would make more sense.

Reply:
I don't think it's a good idea, after all, we can have a search in which we want, for example, to get hints from the collection regarding posts,or followed users